CONTRIBUTING.md

```diff
@@ -131,6 +131,10 @@ There are three ways to setup an Apache Airflow development environment.
   # including the devel which will provide the development tools
   pip install -e '.[hdfs,hive,druid,devel]'
 
+  # Install any other dependencies for specific test groups you might need
+  # e.g.:
+  #       pip install -e '.[gcp]'
```

Comment:
Maybe we should join it with the previous one. I think it would be better to say in the comment above something like "You can also add other dependencies for specific test groups you might need for example gcp_api/s3 etc."

Besides I believe gcp will not work as we have now 'gcp_api' instead .